libcni/api.go

```diff
@@ -28,25 +29,30 @@ type RuntimeConf struct {
 	Args        [][2]string
 }
 
+// NetworkConfig is network configuration network add/del operation
```

Comment:
// NetworkConfig contains data for a network's add/del operation,
// and the data comes from netconf file.